canonical-web-and-design/ubuntu.com
# Display personal contract details

## Done

- Display personal contract details.
- Add fetching of personal subscriptions and types.
- Remove enterprise contracts.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10197.demos.haus/advantage?test_backend=true).
- You should see the [free account details](https://app.zeplin.io/project/60ec5fb4e07cc90fbbf2b318/screen/60edc152de9015ae8a35ccd2) on the right (but not in the tabs at the bottom, that'll be done [later](https://github.com/canonical-web-and-design/commercial-squad/issues/134)).


## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/118.

## Screenshots

<img width="629" alt="Screen Shot 2021-08-27 at 1 31 21 pm" src="https://user-images.githubusercontent.com/361637/131071870-d426d8b4-1810-4d6b-ac57-a346118fc5c6.png">
